src/api/user-data.js

```diff
@@ -27,7 +27,7 @@ export class UserData {
     /** @const {string} */
     this.idToken = idToken;
     /** @private @const {!Object} */
-    this.data = data;
+    this.data_ = data;
```

Comment:
Please keep the name and make it public instead.

Reply:
Done